Source/WebKit/UIProcess/Cocoa/WKTextSelectionRect.h

```diff
@@ -24,18 +24,27 @@
  */
 
 #if PLATFORM(IOS_FAMILY)
-
 #import <UIKit/UIKit.h>
+#elif PLATFORM(MAC)
+#import <pal/spi/mac/NSTextInputContextSPI.h>
+#endif
 
 namespace WebCore {
 class SelectionGeometry;
 }
 
+#if PLATFORM(IOS_FAMILY)
 @interface WKTextSelectionRect : UITextSelectionRect
+#elif PLATFORM(MAC)
+@interface WKTextSelectionRect : NSObject // FIXME: change to NSTextSelectionRect after rdar://126379463 lands
```

Comment:
Nit: Comment convention.
```suggestion
@interface WKTextSelectionRect : NSObject // FIXME: Change to NSTextSelectionRect after rdar://126379463 lands.
```